### PR TITLE
Feature/2433 renew licence reminder

### PR DESCRIPTION
**Pull request by RAnderson96**
Adds a cron job which performs the following

- Sequelize query to find all registrations that will expire in two weeks
- Checks if they are due any returns, if so pass this into the notify email with a list of year[s] due
- Send notify email to applicable licences

Issue: Scottish-Natural-Heritage/Licensing#2433

### CODE REVIEW

File: src/controllers/v2/scheduled.js [hunk 1]
```diff
@@ -169,22 +95,29 @@ const ScheduledController = {
...
+    const fourteenDaysFromNowStart = addDaysSetTime(todaysDate, 14, 0, 0, 0);
+    const fourteenDaysFromNowEnd = addDaysSetTime(todaysDate, 14, 23, 59, 59);
 
+    return Registration.findAll({
+      include: [
+        {model: Return, required: false},
+        {model: Renewal, required: false}
+      ],
+      where: {
+        '$Renewals.id$': {[Op.is]: null},
```
Comment (paulcoxsnh) [inline src/controllers/v2/scheduled.js:109]: Not sure this needs checked? If a registration has been renewed the expiry date will have been updated anyway? And if a registration is renewed will this not fail the following year?

**Review by paulcoxsnh — CHANGES_REQUESTED** [review]
Maybe a small change required...

File: src/controllers/v2/scheduled.js (hunk 1 above)
Comment (RAnderson96) [reply src/controllers/v2/scheduled.js:109]: Good shout Paul, ill make this change to #2434 as well as this uses a similar logic

Comment (paulcoxsnh) [reply src/controllers/v2/scheduled.js]: Resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-api/pull/134/commits/cee901da338ac43bdfed34ffc4ebdfb4996431e7.

**Review by paulcoxsnh — APPROVED** [review]
Looks good :)

**Pull request merged**